packages/stokhos/CMakeLists.txt

```diff
@@ -30,12 +30,8 @@ TRIBITS_ADD_OPTION_AND_DEFINE(${PACKAGE_NAME}_ENABLE_Ensemble_Reduction
   "Enable reduction across across ensemble components."
   OFF )
 
-TRIBITS_ADD_OPTION_AND_DEFINE(
-  Stokhos_ENABLE_CXX11
-  HAVE_STOKHOS_CXX11
-  "Enable C++11 support in Stokhos."
-  "${${PROJECT_NAME}_ENABLE_CXX11}"
-  )
+# FIXME: Remove this hard-coded setting.  We always have C++11 now.
+SET(HAVE_STOKHOS_CXX11 ON)
```

Comment:
Right, someone should refactor Stokhos to not require this var (which I would assume is turned into a macro in a configured header file).